### PR TITLE
VaultPress: Add a note to README that VaultPress is deprecated

### PR DESCRIPTION
**Pull request by pypt**
#### Changes proposed in this Pull Request:

* Add a note to README (visible in wordpress.org) that Jetpack VaultPress is deprecated. Users might get confused between Jetpack VaultPress and Jetpack VaultPress Backup plugins.

#### Other information:

- [ ] ~~Have you written new tests for your changes, if applicable?~~
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

* peaFOp-sK-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* n/a

### CODE REVIEW

File: projects/plugins/vaultpress/readme.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Stable tag: 2.2.0
 Requires PHP: 5.6
 License: GPLv2
 
-Jetpack VaultPress offers real-time backups, one-click restores, and premium support from WordPress experts.
+(DEPRECATED: Please install "Jetpack VaultPress Backup" instead) Jetpack VaultPress offers real-time backups, one-click restores, and premium support from WordPress experts.
```
Comment (Initsogar) [inline projects/plugins/vaultpress/readme.txt:10]: I would suggest using a link pointing to https://wordpress.org/plugins/jetpack-backup/.
Also, I think the idea discussed in peaFOp-sK-p2#comment-319 is to use this message as the short description (removing the other part), could you double-check?

```suggestion
(DEPRECATED: Please install [Jetpack VaultPress Backup](https://wordpress.org/plugins/jetpack-backup/) instead) Jetpack VaultPress offers real-time backups, one-click restores, and premium support from WordPress experts.
```

**Review by Initsogar — COMMENTED** [review]
Thanks for tackling this! I just left a comment. :)

File: projects/plugins/vaultpress/readme.txt (hunk 1 above)
Comment (pypt) [reply projects/plugins/vaultpress/readme.txt:10]: > I would suggest using a link pointing to https://wordpress.org/plugins/jetpack-backup/.

Some fierce googling has revealed that the short description can't have markup:

https://wordpress.org/plugins/readme.txt

> Also, I think the idea discussed in peaFOp-sK-p2#comment-319 is to use this message as the short description (removing the other part), could you double-check?

For now, maybe we'll leave the "(depreciation notice) A short description" form (even if it's more than 150 characters) and see how it goes.

**Review by Initsogar — APPROVED** [review]
Looks good to me!

**Pull request merged**